src/Suggestions/Suggestions.module.css

```
@@ -0,0 +1,115 @@
+.suggestionSection {
+  padding: 112px 0;
+  margin: 0 auto;
+  max-width: 600px;
+  width: 100%;
+  box-sizing: border-box;
+}
+.suggestionSectionOnHomePage {
+  position: fixed;
+  top: 25%;
+  left: 65.45%;
+  height: 100vh;
+  width: 293px;
+}
+.text {
+  font-size: 16px;
+  font-weight: 600;
+  margin: 0;
+  margin-bottom: 6px;
+  margin-left: 13px;
+}
+.textOnHomePage {
+  color: #8e8e8e;
+  font-weight: 600;
+  margin-left: 14px;
+}
+.textViewButton {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.seeAllButton {
+  border: 0;
+  background: none;
+  font-weight: 600;
+  color: #262626;
+  padding: 0;
+  margin-right: 14px;
+  cursor: pointer;
+  text-decoration: none;
+  font-size: 12px;
+}
+.contentSection {
+  background-color: #fff;
+}
+
+.content {
+  display: flex;
+  padding: 8px 16px;
+  align-items: center;
+}
+.contentSectionOnHomePage {
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  padding: 8px 16px;
+  align-items: center;
+}
+.userNameAndReason {
```

The only difference here is `padding: 8px 16px;` and `box-sizing: border-box;`. Why not use sub classes like usual?
```suggestion
.content {
  display: flex;
  padding: 8px 16px;
  align-items: center;
}
.contentSection.onHomePage {
  box-sizing: border-box;
  padding: 8px 16px;
}
```

---

Yea, it turns out there is no difference, so I just gave them the same name.